news/2020-09-15-CDI_Lite_extension.adoc

```diff
@@ -0,0 +1,284 @@
+---
+title: "Build Compatible Extensions Proposal"
+summary: Description of an extension SPI for CDI lite
+layout: post
+authors: [ladt]
+tags: [Proposal]
+desc: Build Compatible Extensions Proposal
+priority: 1
+change_frequency: daily
+---
+
+
+A few months back we shared our vision for CDI lite.
+To make it short the goal with CDI lite is to make the spec cloud native. This will be done by identifying a features' subset that can be implemented with a build time only approach (no more runtime proxies and other “magic” requirements) .
```

Comment:
_"to make the spec cloud native"_ - this is not a marketing leaflet. Could we try to avoid similar formulations?